schemas/dbt/manifest/v12.json

```diff
@@ -19098,6 +19098,23 @@
                               "type": "null"
                             }
                           ]
+                        },
+                        "order_by": {
```

Comment:
Not sure if you saw this / used it but there is a `make json_schema` target you can use to generate this.